src/corner-dialog/src/CornerDialog.js

```diff
@@ -130,7 +128,12 @@ export default class CornerDialog extends PureComponent {
     /**
      * Props that are passed to the dialog container.
      */
-    containerProps: PropTypes.object
+    containerProps: PropTypes.object,
+
+    /**
+     * Props that will set position of corner dialog
+     */
+    position: PropTypes.object
```

Comment:
I think we should do like we do in other components and only take a specific set of constants instead of an object.

We will apply the right css based on that: e.g. `position={POSITIONS.TOP_LEFT}`. That way we can prevent weird scenarios like vertically and horizontally centered _corner_ dialogs, etc.

Thoughts?

Reply:
I will change this!